backend/globaleaks/handlers/base.py

```diff
@@ -69,7 +69,8 @@ def connection_check(tid, role, client_ip, client_using_tor):
     :param client_ip: A client IP
     :param client_using_tor: A boolean for signaling Tor use
     """
-    ip_filter_enabled = State.tenants[tid].cache.get('ip_filter_' + role + '_enable')
+    ip_filter_enabled = State.tenants[tid].cache.get(
```

Comment:
@joeman65 @alessiofranceschini: This seems to be just stilistic  code review, please provide this separately to not misk code review of our code with the features you need to implement.